src/cell/mod.rs

```diff
@@ -36,3 +36,16 @@
 //!
 //! There are two types of wall cells: *constrained* and *unconstrained* ones.
 //! *Constrained* wall cells contain a number from `0` to `3`.
+
+// Exported Types
+pub use self::cell::Cell;
```

Comment:
Move each of these `use` statements to the commits that intoduce the corresponding module.